Server/mods/deathmatch/logic/luadefs/CLuaAccountDefs.cpp

```diff
@@ -513,6 +515,17 @@ int CLuaAccountDefs::RemoveAccount(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaAccountDefs::SetAccountSerial(std::string strName, std::string strSerial)
```

Comment:
Please avoid using Hungarian notation in new code, `name` and `serial` would be neater

Reply:
adjusted 9ad1188